t_sites/test_tempesta_tech.py

```diff
@@ -0,0 +1,315 @@
+            "load_cookies": True,
+        },
+        {
+            "id": "login",
+            "type": "curl",
+            "http2": True,
+            "save_cookies": True,
+            "uri": "/wp-login.php",
+        },
+        {
+            "id": "purge_cache",
+            "type": "curl",
+            "http2": True,
+            # Set max-time to prevent hang caused by Tempesta FW #1692
+            "cmd_args": "--request PURGE --max-time 1",
+        },
+        {
+            "id": "nghttp",
+            "type": "external",
+            "binary": "nghttp",
+            "cmd_args": (
+                " --no-verify-peer"
+                " --get-assets"
+                " --null-out"
+                " --header 'Cache-Control: no-cache'"
+                " https://${tempesta_ip}"
+            ),
+        },
+    ]
+
+    tempesta = {
+        "config": """
+            listen 443 proto=h2;
+
+            cache 2;
+            cache_fulfill * *;
+            cache_methods GET HEAD;
+            cache_purge;
+            # Allow purging from the containers (upstream), localhost (VM) and the host.
+            cache_purge_acl ${server_ip} 127.0.0.2;
+
+            access_log on;
+
+            frang_limits {
+                request_rate 200;
+                http_method_override_allowed true;
+                http_methods post put get purge;
+            }
+
+            block_action attack reply;
+            block_action error reply;
+
+            # Make WordPress to work over TLS.
+            # See https://tempesta-tech.com/knowledge-base/WordPress-tips-and-tricks/
+            req_hdr_add X-Forwarded-Proto "https";
+
+            resp_hdr_set Strict-Transport-Security "max-age=31536000; includeSubDomains";
+
+            tls_certificate ${tempesta_workdir}/tempesta.crt;
+            tls_certificate_key ${tempesta_workdir}/tempesta.key;
+
+            srv_group default {
+                    server ${server_ip}:8000;
+
+            }
+            vhost default {
+                    tls_match_any_server_name;
+                    proxy_pass default;
+            }
+
+            http_chain {
+                # Redirect old URLs from the old static website
+                uri == "/index"		-> 301 = /;
+                uri == "/development-services" -> 301 = /network-security-performance-analysis;
+
+                # Proably outdated redirects
+                uri == "/index.html"	-> 301 = /;
+                uri == "/services"	-> 301 = /development-services;
+                uri == "/services.html"	-> 301 = /development-services;
+                uri == "/c++-services"	-> 301 = /development-services;
+                uri == "/company.html"	-> 301 = /company;
+                uri == "/blog/fast-programming-languages-c-c++-rust-assembly" -> 301 = /blog/fast-programming-languages-c-cpp-rust-assembly;
+
+                    -> default;
+            }
+        """
+    }
+
+    def get_response(self, client: CurlClient) -> CurlResponse:
```

Comment:
Just style. As for me, `get_response` indicates getter method or to access something in very short time. In this case I would prefer to use `send_request` or something similar which describes that some of work will be done(network transfer) before getting response.

Reply:
Done